profiler/options.go

```diff
@@ -34,12 +34,14 @@ const (
 	// For more information or for changing this value, check MutexProfileFraction
 	DefaultMutexFraction = 10
 
-	// DefaultBlockRate specifies the default block profiling rate used by the
-	// block profiler. For more information or for changing this value, check
-	// BlockProfileRate. The default rate is chosen to prevent high overhead
-	// based on the research from:
-	// https://github.com/felixge/go-profiler-notes/blob/main/block.md#benchmarks
-	DefaultBlockRate = 10000
+	// DefaultBlockRate specifies the default block profiling rate (in ns) used
+	// by the block profiler. For more information or for changing this value,
+	// check BlockProfileRate(). The default value of 100ms is somewhat
+	// arbitrary. There is no provably safe value that will guarantee low
+	// overhead for this profile type for all workloads. We don't recommend
+	// enabling it under normal circumstances. See the link below for more
+	// information: https://github.com/DataDog/go-profiler-notes/pull/15/files
+	DefaultBlockRate = 100_000_000
```

Comment:
Underscores in numeric literals don't compile for Go 1.12 :( I believe this is also enforced by our go.mod file.

Reply:
Thanks, will fix.

![](https://media.giphy.com/media/3xz2BLBOt13X9AgjEA/giphy.gif)